[Hono] Device Registry with Embedded DB does not support Kafka

The embedded device registry fails to start when Kafka-based messaging is enabled with the following exception:

```
15:13:17.173 [main] ERROR o.e.h.d.jdbc.Application - exception occurred during startup, shutting down ...
java.util.concurrent.ExecutionException: java.lang.IllegalStateException: Event client must be set
        at java.base/java.util.concurrent.CompletableFuture.reportGet(Unknown Source)
        at java.base/java.util.concurrent.CompletableFuture.get(Unknown Source)
        at org.eclipse.hono.service.spring.AbstractBaseApplication.run(AbstractBaseApplication.java:163)
        at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:795)
        at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:785)
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:333)
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:1311)
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:1300)
        at org.eclipse.hono.deviceregistry.jdbc.Application.main(Application.java:52)
Caused by: java.lang.IllegalStateException: Event client must be set
        at org.eclipse.hono.deviceregistry.service.device.AutoProvisioner.start(AutoProvisioner.java:153)
        at org.eclipse.hono.deviceregistry.service.device.AbstractRegistrationService.lambda$start$0(AbstractRegistrationService.java:71)
        at io.vertx.core.Future.lambda$compose$3(Future.java:368)
        at io.vertx.core.impl.FutureImpl.dispatch(FutureImpl.java:105)
        at io.vertx.core.impl.FutureImpl.tryComplete(FutureImpl.java:150)
        at io.vertx.core.impl.FutureImpl.complete(FutureImpl.java:111)
        at io.vertx.core.impl.FutureImpl.handle(FutureImpl.java:176)
        at io.vertx.core.impl.FutureImpl.handle(FutureImpl.java:21)
        at io.vertx.core.impl.FutureImpl.dispatch(FutureImpl.java:105)
        at io.vertx.core.impl.FutureImpl.tryComplete(FutureImpl.java:150)
        at io.vertx.core.impl.FutureImpl.complete(FutureImpl.java:111)
        at io.vertx.core.impl.FutureImpl.handle(FutureImpl.java:176)
        at io.vertx.core.impl.FutureImpl.handle(FutureImpl.java:21)
        at io.vertx.core.impl.FutureImpl.dispatch(FutureImpl.java:105)
        at io.vertx.core.impl.FutureImpl.tryComplete(FutureImpl.java:150)
        at io.vertx.core.impl.FutureImpl.complete(FutureImpl.java:111)
        at io.vertx.core.impl.FutureImpl.handle(FutureImpl.java:176)
        at io.vertx.core.impl.FutureImpl.handle(FutureImpl.java:21)
        at io.vertx.ext.jdbc.impl.JDBCClientImpl.lambda$doClose$12(JDBCClientImpl.java:356)
        at io.vertx.core.impl.ContextImpl.lambda$null$0(ContextImpl.java:327)
        at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:366)
        at io.vertx.core.impl.EventLoopContext.lambda$executeAsync$0(EventLoopContext.java:38)
        at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
        at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:472)
        at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
        at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.base/java.lang.Thread.run(Unknown Source)
15:13:17.248 [main] INFO  o.e.h.d.jdbc.Application - shutting down application...
```

--- a/charts/hono/Chart.yaml
+++ b/charts/hono/Chart.yaml
@@ -15,7 +15,7 @@ name: hono
 description: |
   Eclipse Hono™ provides remote service interfaces for connecting large numbers of IoT devices to a back end and
   interacting with them in a uniform way regardless of the device communication protocol.
-version: 1.9.1
+version: 1.9.2
 # Version of Hono being deployed by the chart
 appVersion: 1.8.0
 keywords:

--- a/charts/hono/templates/hono-service-device-registry-embedded/hono-service-device-registry-secret.yaml
+++ b/charts/hono/templates/hono-service-device-registry-embedded/hono-service-device-registry-secret.yaml
@@ -72,11 +72,12 @@ stringData:
           management:
             url: "jdbc:h2:/var/lib/hono/device-registry/registry"
             driverClass: "org.h2.Driver"
-      {{- include "hono.amqpMessagingNetworkClientConfig" $args | nindent 6 }}
+      {{- include "hono.messagingNetworkClientConfig" $args | nindent 6 }}
       {{- include "hono.healthServerConfig" .Values.deviceRegistryExample.hono.healthCheck | nindent 6 }}
 data:
   key.pem: {{ .Files.Get "example/certs/device-registry-key.pem" | b64enc }}
   cert.pem: {{ .Files.Get "example/certs/device-registry-cert.pem" | b64enc }}
   trusted-certs.pem: {{ .Files.Get "example/certs/trusted-certs.pem" | b64enc }}
   auth-server-cert.pem: {{ .Files.Get "example/certs/auth-server-cert.pem" | b64enc }}
+  truststore.jks: {{ .Files.Get "example/certs/trustStore.jks" | b64enc }}
 {{- end }}

